sites/fast-website/src/app/components/section-header/section-header.template.ts

```diff
@@ -0,0 +1,11 @@
+import { html } from "@microsoft/fast-element";
+import { SectionHeader } from "./section-header";
+
+export const SectionHeaderTemplate = html<SectionHeader>`
+    <template>
+        <slot name="heading"></slot>
+        <p part="short">${x => x.short}</p>
+        <p part="body">${x => x.body}</p>
```

Comment:
Are we going to add styles for the body here? We'll want to add and target a class if so.